content/millennium/r4/financial/general/charge-item.md

```diff
@@ -61,11 +61,11 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  `description`          | [`string`]                        | A description providing additional details of the resource.
  `modifier`             | None (contains nested extensions) | A code providing additional detail about a product or service.
  `net-price`            | [`Money`]                         | The quantity times the unit price for a resource (total price).
- `offset-by`            | [`Reference`]                     | Indicates a resource that this resource is offset by. This resource is no longer active when offset.
- `performing-location`  | [`Reference`]                     | A location where the resource was performed.
+ `offset-by`            | [Reference]                       | Indicates a resource that this resource is offset by. This resource is no longer active when offset.
+ `performing-location`  | [Reference]                       | A location where the resource was performed.
  `priority`             | [`unsignedInt`]                   | The priority of the element within a list.
  `procedure`            | None (contains nested extensions) | Procedure performed on the patient associated to the resource.
- `replacing`            | [`Reference`]                     | A reference to a resource that this resource is replacing.
+ `replacing`            | [Reference]                       | A reference to a resource that this resource is replacing.
```

Comment:
What was the reason for changing this? It is [`Reference`] for other resources.

Reply:
We needed multiple reference links(as one is Object and the other is just search type) 
I referring the object with [Reference] and type as [`reference`]. I couldn't have both as the type as the parser is only not case-sensitive and is treating `Reference` and `reference` the same.

Reply:
there is only one Reference to that link. think we have multiple references to that object extracting it as a variable looks better.

Reply:
I think for the capitalized `[Reference]` fields, let's try to implement the hotlinks the way Keifer suggested here: https://github.com/cerner/fhir.cerner.com/blame/main/content/millennium/r4/base/management/encounter.md#L68

We can leave the lowercase hotlink implementation untouched similar to what the Encounter resource is doing already.

Reply:
addressed: 4f6b0836109ab36d830dac57bcc5b24712ff147b